docs/index.rst

```diff
@@ -11,24 +11,33 @@ hardware.
 Quick start
 -----------
 
-Define a circuit that generates 2-qubit Bell state and execute it on a simulator backend:
+Define a circuit that generates 2-qubit Bell state and sample it on a simulator backend running on the local machine:
 
 .. jupyter-execute::
 
-   import qiskit
    from qiskit import QuantumCircuit
+
    from qiskit_aqt_provider import AQTProvider
+   from qiskit_aqt_provider.primitives import AQTSampler
+
+   # Define a circuit
+   circuit = QuantumCircuit(2)
+   circuit.h(0)
+   circuit.cnot(0, 1)
+   circuit.measure_all()
 
-   backend = AQTProvider("ACCESS_TOKEN").get_backend("offline_simulator_no_noise")
+   # Select an execution backend
+   provider = AQTProvider("ACCESS_TOKEN")
```

Comment:
We can consider calling the ACCESS_TOKEN something else (maybe adding DUMMY) so that it is clear that this value is not relevant for this example, i.e. for the local simulator. Alternatively we can also add a comment/hint that explains that this can be an arbitrary string.

Reply:
How about 94e842e8afc72be786d8646d86daffadbe453c00?

Reply:
fine with me :+1: